domehunter/dome_control.py

```diff
@@ -319,31 +353,28 @@ def goto_az(self, az):
         """
         if self.dome_az is None:
             return
-
-        target_az = Longitude(az * u.deg)
-        logger.debug(f'Goto target azimuth: {target_az}')
+        if self._move_event.is_set():
+            logger.debug('Movement command in progress')
+            raise Exception("Movement command in progress")
```

Comment:
Not sure if you actually want to raise an exception because if that's not handled properly by the user then the code can stop running and that might have side effects. Maybe just change to `logger.warning` and do a `return` (or `return False`).